init.lua

```diff
@@ -5,6 +5,8 @@ require('cudnn.ffi')
 local C = cudnn.C
 local ffi = require 'ffi'
 
+local thc = ffi.load("THC")
```

Comment:
this (and the change below) needs to be a windows-only change. can you guard it to be so.
The rest of the PR looks good.